docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -89,7 +89,8 @@ Nuxt 3 starting from rc.12 comes with the public beta for route rules and hybrid
 - `ssr` - Disables server-side rendering for sections of your app and make them SPA-only with `ssr: false`
 - `cors` - Automatically adds cors headers with `cors: true` - you can customize the output by overriding with `headers`
 - `headers` - Add specific headers to sections of your site - for example, your assets
-- `static` and `swr` - `static` enables a single (on-demand) build; `swr` enables a static build, that lasts for a configurable TTL. (currently enables full incremental static generation on Netlify, with Vercel coming soon)
+- `swr`: Server-side renders sections of your app and add cache headers in the response for the server or the reverse proxy to cache it for a configurable TTL.
```

Comment:
On Netlify & Vercel this uses their output APIs (Builder Functions and Build Output API, respectively) to add it to the CDN layer (so it's not just adding cache headers). And within the node server preset this caches the response _within_ the server, so it's not just adding cache headers there either.

Reply:
Sure I can change the documentation accordingly then,